src/Data/Widget/Adapters/MetricAdapter.elm

```diff
@@ -8,6 +8,9 @@ import Dict exposing (Dict)
 import Json.Decode as Json exposing (Value)
 
 
+-- TODO These are flipped (y, x) and zero based which is unique amongst the adapters
+
```

Comment:
D'oh!

Reply:
To be fair, I think you did this first and I broke rank. But (x, y) makes more sense to me. The zero/one based we could argue forever ;) 

Did confuse me a bit until I unpicked it tho.

What I'd really like, having now spent some time writing config for these widgets, is to be able to express the range same as Sheets e.g. A1:L9 

Rainy day job.

Reply:
`A1:L9` 👍 🥇 

(x,y) Also makes most sense to me as well, I must have been having a moment.  

"Along the corridor and up the stairs" and all that.